test.sh

```diff
@@ -3,5 +3,9 @@ set -euo pipefail
 
 yarn install
 yarn prod
-yarn test
+yarn coverage
 yarn lint
+
+curl -Os https://uploader.codecov.io/latest/linux/codecov
+chmod +x codecov
+./codecov -t ${CODECOV_TOKEN} --dir ./coverage
```

Comment:
Small nit, typically we want to end with a blank line so the next diff of anything added doesn't say it modified this line.